test/integration-test.py

```diff
@@ -241,6 +242,42 @@ def run_client_tests():
     cmd = os.path.join(root, 'tests', 'boulder-integration.sh')
     run_custom(cmd, cwd=root)
 
+
+def get_future_output(cmd, date, cwd=None):
+    return subprocess.check_output(cmd, cwd=cwd, env={'FAKECLOCK': date.isoformat()}, shell=True)
+
+def run_expired_authz_purger_test(challenge_types):
+    subprocess.check_output('''node test.js --email %s --agree true --domains %s --abort-step %s''' % ("purger@test.com", "eap-test.com", "startChallenge"), shell=True)
```

Comment:
This line should actually be split at some reasonable place. I realize I said "merge abort-step onto previous line," which was a little ambiguous. More precisely, we should either use one line per flag, or (my preference here), try to fill up the lines to some degree. The code I was commenting on had a bit of a mix.